e2e/cypress/support/commands.ts

```diff
@@ -30,26 +30,20 @@ Cypress.Commands.add('findAdSlotIframeBySlotId', (adSlotId: string) => {
 	cy.get(`#${adSlotId}`).find('iframe', { timeout: 30000 });
 });
 
-const allowAllButtons = ['Yes, I’m happy', 'Accept all', 'Yes, ok']
-	.map((title) => `button[title="${title}"]`)
-	.join(',');
-const manageConsent = 'Manage my cookies';
-const rejectAll = 'Reject all';
-
 Cypress.Commands.add('rejectAllConsent', () => {
 	cy.getIframeBody('sp_message_iframe_')
-		.find(`button[title="${manageConsent}"]`)
+		.find('button.sp_choice_type_12')
```

Comment:
Could we add a comment here to explain that this is Sourcepoint's special class name for the "manage cookies" button? And also maybe that this is the currently the best way available to us to identify this button.